tests/Unit/Data/GraphiteStoreTest.php

```diff
@@ -41,12 +42,14 @@ protected function setUp(): void
 
         // fix the date
         Carbon::setTestNow(Carbon::createFromTimestamp($this->timestamp));
+        Config::set('graphite.enable', true);
```

Comment:
@murrant does this look like a good fix to you?

else the "isEnabled()" check will fail inside Graphite and the createClient will never run.

Reply:
thats probably fine.  I think config should be reset between tests now, but maybe the cache interferes with that.